ts/features/bonus/bpd/screens/details/components/summary/textualSummary/ClosedTextualSummary.tsx

```diff
@@ -84,14 +114,7 @@ const OK = (props: Props) => (
           amount: formatNumberAmount(props.period.amount.totalCashback)
         }
       )}
-      {/* If the max amount is reached, inform the user */}
-      {props.period.amount.totalCashback >= props.period.maxPeriodCashback
-        ? ": " +
-          I18n.t(
-            "bonus.bpd.details.components.transactionsCountOverview.closedPeriodMaxAmount"
-          )
-        : "!"}
-      {"\n"}
+      {enhanceOkText(props).getOrElse("") + "!\n"}
```

Comment:
```suggestion
      {enhanceOkText(props)
        .map(t => t + "!\n")
        .getOrElse("")}
```
Is the `none` case possibile? if yes could the suggestion prevent we display a single question point?

Reply:
Yes, the `none` case is possible (the user earn a cashback amount < max) and we should have always the final `!\n`, also for the amount < max.